tests/test_cli.py

```diff
@@ -307,7 +307,7 @@ def test_clean(mock_path, tmp_path, args, expected):  # pylint: disable=unused-a
 @pytest.mark.parametrize(
     "build_args,export_args,expected_tables",
     [
-        (["build", "-t", "core"], ["export", "-t", "core"], 49),
+        (["build", "-t", "core"], ["export", "-t", "core"], 57),
```

Comment:
as this number creeps higher and higher, i'm socializing [this thought](https://laughingman7743.github.io/PyAthena/cursor.html#asynccursor) that's been rattling around in the back of my head for a bit.